src/utils/attachments.ts

```diff
@@ -0,0 +1,37 @@
+import { array, flatten } from "fp-ts/lib/Array";
+import { task, Task } from "fp-ts/lib/Task";
+import { MessageAttachment } from "generated/backend/MessageAttachment";
+import { PrescriptionData } from "generated/backend/PrescriptionData";
+import { toBarcode } from "./barcode";
+
+const MIME_TYPES = {
+  png: "image/png",
+  svg: "image/svg+xml"
+};
+
+const toBarcodeAttachments = (name: string, value: string) => {
```

Comment:
can you add a comment to this method to explain what `name` and `value` represents ?

Reply:
what's the inferred return type of this method ? (it should be `MessageAttachment[]`, if not you must specify it)